litmus-portal/k8s-manifest.yml

```diff
@@ -75,8 +75,12 @@ spec:
               value: "true"
             - name: SUBSCRIBER_IMAGE
               value: "litmuschaos/litmusportal-subscriber:ci"
+            - name: SUBSCRIBER_NAMESPACE
```

Comment:
I think we can go for only 1 ns? The user will be deploying the self-deployer and subscriber components in the same ns?

Reply:
Let it be seperate for now. We have to make a common namespace var for argo + litmus-portal + litmus components